pep-0692.rst

```diff
@@ -539,10 +508,10 @@ previous example::
 To accomplish this, we propose a new dunder called ``__typing_kwargs_unpack__``.
 The double asterisk syntax should result in a call to the
 ``__typing_kwargs_unpack__`` special method on an object it was used on.
-This means that ``def foo(**kwargs: **T): ...`` is equivalent to
+This means that at runtime, ``def foo(**kwargs: **T): ...`` is equivalent to
 ``def foo(**kwargs: T.__typing_kwargs_unpack__()): ...``.
-``TypedDict`` is the only type in the standard library that will implement
-``__typing_kwargs_unpack`` and it should return ``Unpack[self]``.
+``TypedDict`` is the only type in the standard library that is expected to
+implement ``__typing_kwargs_unpack__``, which should return ``Unpack[self]``.
```

Comment:
This also feels like needless duplication.

Reply:
Kept this sentence here and removed it from the Abstract instead - in my opinion it fits this section better (there's more context around introducing a new dunder) and for abstracts I like to include as little detail as possible.